rasa/core/brokers/pika.py

```diff
@@ -146,7 +150,9 @@ async def connect(self) -> None:
         logger.info(f"RabbitMQ connection to '{self.host}' was established.")
 
         channel = await self._connection.channel()
-        logger.debug("RabbitMQ channel was opened. Declaring fanout exchange.")
+        logger.debug(
+            f"RabbitMQ channel was opened. Declaring fanout exchange. exchange name: {self.exchange_name}"
```

Comment:
what do you think?
```suggestion
            f"RabbitMQ channel was opened. Declaring fanout exchange '{self.exchange_name}'."
```

Reply:
Yeah make sense. Will make changes as suggested